javascript-constructors/example-constructor.js

```diff
@@ -0,0 +1,10 @@
+function ExampleConstructor() {
+
+}
+console.log('ExampleConstructor.prototype.value : ', ExampleConstructor.prototype.value);
```

Comment:
This should just be `ExampleConstructor.prototype`, no need for the `value` property. It's just asking for what the `prototype` output is, not accessing any `value` property.